hudi-common/src/main/java/org/apache/hudi/common/model/DefaultHoodieRecordPayload.java

```diff
@@ -97,4 +86,20 @@ public DefaultHoodieRecordPayload(Option<GenericRecord> record) {
     }
     return metadata.isEmpty() ? Option.empty() : Option.of(metadata);
   }
+
+  protected boolean noNeedUpdatePersistedRecord(IndexedRecord currentValue,
```

Comment:
would rename the method name to needUpdatePersistedRecord?

Reply:
Yeah, needUpdatePersistedRecord looks more read able